src/System.Web.Http/ModelBinding/ErrorParameterBinding.cs

```diff
@@ -20,7 +20,7 @@ public ErrorParameterBinding(HttpParameterDescriptor descriptor, string message)
         {
             if (message == null)
             {
-                throw Error.ArgumentNull(message);
+                throw Error.ArgumentNull("message");
```

Comment:
Should this be `nameof(message)` instead?
@dougbu  you're probably more familiar with the convention used in this repo.

Reply:
We don't use `nameof` at all in this repository. Most of the code here predates C# 6.